card/lib/generators/card/USAGE

```diff
@@ -0,0 +1,21 @@
+Description
+
+Examples
+  wagn generate card new cards pre_fix
```

Comment:
> Should we copy the rake tasks into the new namespace and have two for a while?

Lord no!  Stay thirsty and DRY, my friend. The tasks should be defined in wagn / card and loaded into decko-rails as needed.

> later just decko
imo, always decko-rails.  we don't want to imply that this usage is what *decko* is. Nor do we want decko to be burdened with supporting a special plug-in case.

> Generate a migration to add cards to my app

I don't think we want to mess with their migrations; let's just use our own.   We're not going to add migrations to their app every time we update our structure.  We already have 2 custom migrations tables in our app case (card and deck); in the engine case we should probably have 3, to include the card_structure.

> Follow the instructions output from step 3.
> Edit config/routes.rb, config/application.rb and Rakefile

are these two the same?  I would think that only config/routes.rb would be necessary.  There's no mention of either of these two in the engine guide.

> Run migrations: rake db:migrate
> Load card seeds: rake decko:seed

So many commands.  We're going to be exhausting folks.

> Run tests: rake decko:rspec

You're not saying that running tests is part of standard installation are you?  I don't think I've ever done that with any software before. This would mean they'd have to install many more dependencies, and would dramatically up the odds of problems.

How's this:
1. add 'decko-rails' to Gemfile of existing rails app
2. `bundle install`
3. `decko-rails new wiki`.
4. follow output instructions to edit config
5. `decko-rails seed`

This mirrors our normal installation process very closely; in fact, you can drop the "-rails" and it's basically the app deck commands.